addon/components/time-series-chart.js

```diff
@@ -574,10 +573,10 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
       d3.select(element).classed('hovered', true);
 
       var time = data.labelTime != null ? data.labelTime : data.time;
+      time = this.adjustTimeForShowDetails(time);
```

Comment:
I didn't want the function to have to know about the many places and fallback data of the chart.  It seems cleaner and more unit friendly to simply take a date and return a date.